opentreemap/manage_treemap/templates/manage_treemap/management_base.html

```diff
@@ -0,0 +1,88 @@
+        <div  class="management-container">
+            <div class="management-sidebar tab-list" id="tab-list">
+                <table class="map-switcher">
+                    <tbody>
+                    <tr>
+                        <td class="brand-icon-container">
+                            <div class="brand-icon">
+                                <i class="icon-tree"></i>
+                            </div>
+                        </td>
+                        <td>
+                            <div class="full-section brand">
+                                {{ request.instance.name }}
+                            </div>
+                        </td>
+                    </tr>
+                    </tbody>
+                </table>
+                <table class="nav nav-list">
+                    <tbody>
+                    {% block extra_menu_items %}
+                    {% endblock extra_menu_items %}
+                    </tbody>
+                </table>
+            </div>
+
+            <div class="management-content tab-content">
+                {% block tab_content %}
+                {% endblock tab_content %}
+            </div>
+        </div>
+    </div>
+{% endblock content %}
+
+{% block templates %}
+    {{ block.super }}
+
+    {# No search on management pages #}
+    {% block searchscripts %}
+    {% endblock searchscripts %}
+
+    {% verbatim %}
+    <script id="field-error-template" type="text/x-mustache-template">
+        <ul class="list-unstyled">
+            {{#errors}}
+            <li>{{field}}: {{message}}</li>
+            {{/errors}}
+        </ul>
+    </script>
+
+    <script id="global-errors-template" type="text/x-mustache-template">
+        <ul class="list-unstyled">
+            {{#globalErrors}}
+            <li>{{.}}</li>
+            {{/globalErrors}}
+        </ul>
+    </script>
+    {% endverbatim %}
+{% endblock templates %}
+
+{% block head_extra %}
+    <!-- Start of opentreemap Zendesk Widget script -->
+    <script>/*<![CDATA[*/window.zEmbed||function(e,t){var n,o,d,i,s,a=[],r=document.createElement("iframe");window.zEmbed=function(){a.push(arguments)},window.zE=window.zE||window.zEmbed,r.src="javascript:false",r.title="",r.role="presentation",(r.frameElement||r).style.cssText="display: none",d=document.getElementsByTagName("script"),d=d[d.length-1],d.parentNode.insertBefore(r,d),i=r.contentWindow,s=i.document;try{o=s}catch(c){n=document.domain,r.src='javascript:var d=document.open();d.domain="'+n+'";void(0);',o=s}o.open()._l=function(){var o=this.createElement("script");n&&(this.domain=n),o.id="js-iframe-async",o.src=e,this.t=+new Date,this.zendeskHost=t,this.zEQueue=a,this.body.appendChild(o)},o.write('<body onload="document._l();">'),o.close()}("https://assets.zendesk.com/embeddable_framework/main.js","opentreemap.zendesk.com");/*]]>*/</script>
+    <!-- End of opentreemap Zendesk Widget script -->
+{% endblock head_extra %}
```

Comment:
ZenDesk includes the ability for people to create support tickets with us, and also includes help articles for billing / changing your plan, so I think we'll want to keep the ZenDesk link in the closed source addons.